### DISCUSSION TITLE
```scratchr2``` forum button fix

### DISCUSSION THREAD

**Discussion by Mbrick2** [ScratchAddons/ScratchAddons#4258] [Ideas]
I don't trust my self to PR but here is what I found
## How it looks
![image](https://user-images.githubusercontent.com/90691415/153357522-f6fcca7c-296d-4a12-84f8-f4b891c0e977.png)
### Previous/Next Buttons
![Screen Shot 2022-02-10 at 6 24 18  pm](https://user-images.githubusercontent.com/90691415/153357883-1af2bc3f-0c3b-4ebd-81f5-7d59fe6ba768.png)
### Other Buttons
![Screen Shot 2022-02-10 at 6 24 37 pm](https://user-images.githubusercontent.com/90691415/153357983-8338a6d3-d1b8-45e9-8d70-c78c7aa661f7.png)
### Pagination 
![Screen Shot 2022-02-10 at 6 24 18  pm](https://user-images.githubusercontent.com/90691415/153358159-4f94a95a-ac3e-4b7c-b9f1-1557711cfb12.png)

**Comment by GrahamSH-LLK** [comment]
What is the problem? the fact that the pagination related buttons are longer?

**Reply by WorldLanguages** [reply]
Yeah @Mbrick2 can you clarify and post a full screen screenshot?

**Reply by CST1229** [reply]
I think this is a suggestion to tweak the buttons, not a bug report.

**Reply by WorldLanguages** [reply]
Title says "fix" 🤔

**Reply by CST1229** [reply]
The category is Ideas.

**Reply by Mbrick2** [reply]
> The category is Ideas.

Correct

> Title says "fix" 🤔

Sorry

> Yeah @Mbrick2 can you clarify and post a full screen screenshot?

![image](https://user-images.githubusercontent.com/90691415/153357522-f6fcca7c-296d-4a12-84f8-f4b891c0e977.png)
 it looks more '3.0'

**Reply by mxmou** [reply]
> it looks more '3.0'

I don't think so. The page numbers should be circular, and making a button bigger uses more space but doesn't automatically make it more 3.0.
